charts/server/README-sub.md.gotmpl

```diff
@@ -1,16 +1,11 @@
 {{ template "chart.header" . }}
+
```

Comment:
Shouldn't this file stay as Readme.md.gotmpl now?

Reply:
Ah I wanted to remove it entirely - the existing file doesn't do much different than the overall default (which is the same as not having a template at all).